src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -2274,9 +2274,9 @@ impl Step for LlvmTools {
 
         let target = self.target;
 
-        /* run only if llvm-config isn't used */
+        // Run only if a custom llvm-config is not used
         if let Some(config) = builder.config.target_config.get(&target) {
-            if let Some(ref _s) = config.llvm_config {
+            if !builder.config.llvm_from_ci && config.llvm_config.is_some() {
```

Comment:
Running any `dist` command when CI LLVM is enabled should IMO be a hard error. But that's a separate topic.